src/module/Todo.js

```diff
@@ -0,0 +1,7 @@
+export default class Todo {
+  constructor(todo, completed, index) {
+    this.description = todo;
+    this.completed = completed;
+    this.index = index;
```

Comment:
- [ ] When I start to delete tasks, the indexs start getting messed up. They should remain in order 1,2,3...
![image](https://user-images.githubusercontent.com/62705947/199725791-e6fdd620-db20-4d78-b875-8e51ecedf8df.png)